tools/lint/constants.py

```diff
@@ -0,0 +1,22 @@
+
+CSS_REQUIREMENT_FLAGS = {
+    'valid': frozenset({
+        'asis',
+        'HTMLonly',
+        'invalid',
+        'may',
+        'nonHTML',
+        'paged',
+        'scroll',
+        'should',
+        'svg'
+    }),
+    'deprecated': frozenset({
```

Comment:
How many do we have of these? Can they be removed instead of adding them to the lint?

Reply:
Based on [this run](https://community-tc.services.mozilla.com/tasks/IDor1Du-RimCuMg0BDuAog/runs/0/logs/public/logs/live.log) we have:
`INFO:lint:There were 6907 errors (DEPRECATED-CSS-REQUIREMENT-FLAG: 1066 INVALID-CSS-REQUIREMENT-FLAG: 5841)`
We can begin removing the invalid or deprecated flags as well. If we remove the existing flags, would it not be worth keeping a check in the lint to make sure no more invalid flags are added?